### PR TITLE
Add no css example to readme

### PR DESCRIPTION
**Pull request by GeeCastro**
Adresses #100 in the readme

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -174,6 +174,21 @@ change:
         return ImageBlockConverter(**options).convert(html)
 
 
+    from markdownify import MarkdownConverter
```
Comment (aytey) [inline README.rst:177]: Wow, looks great!

Minor nit: these should be two separate blocks; as they are currently, they run in to eachother suggesting they're "one" solution.

Comment (Chichilele) [reply README.rst:177]: I agree with you! Should we have 2 examples at all @matthewwithanm? If yes, is there a way to split the code blocks with the indented syntax? I usually use the fenced suntax 😄

Comment (aytey) [reply README.rst:177]: I haven't tried, but what happens if you just do a newline between the blocks and start the newly-added line with:

```
.. code:: python
```

?

**Comment by chrispy-snps** [comment]
@AlexVonB, @GeeCastro - now that `<script>` and `<style>` are ignored by the base code per #112, I think we should remove this workaround example as it would no longer have any effect. Are you agreeable to this?

**Comment by chrispy-snps** [comment]
@GeeCastro - if it weren't for the unfortunately-timed (?) fix, this would have been a great example of the extension mechanism! Thank you for contributing.

**Comment by GeeCastro** [comment]
I think the doc could be clearer if it said how easy the `def concert_<tag>(…)` works. Let me open another small pr

**Comment by GeeCastro** [comment]
#116 something like this feel free to rephrase

**Pull request merged**